log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -95,6 +104,11 @@ private Logging()
         redirectStdStreams();
     }
 
+    public void registerJmxExports(MBeanExporter exporter)
```

Comment:
add a comment stating that logging typically is initialized before JMX so export must be lazy